dependencies.yaml

```diff
@@ -352,7 +352,7 @@ dependencies:
               - *libcusparse114
   cuda_wheels:
     specific:
-      - output_types: pyproject
+      - output_types: [pyproject, requirements]
```

Comment:
This should be fine... any builds that don't want these (i.e. devcontainers) pass `use_cuda_wheelse="false"`, so get the empty matrix a few lines down.

ref: https://github.com/rapidsai/devcontainers/blob/a6c9100b88160bc62072a358613c28193038bc4f/features/src/rapids-build-utils/opt/rapids-build-utils/bin/make-pip-dependencies.sh#L86